src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb

```diff
@@ -50,6 +50,14 @@ Public Module BuildDevDivInsertionFiles
             "codeAnalysisService.servicehub.service.json",
             "remoteHostService.servicehub.service.json",
             "serviceHubSnapshotService.servicehub.service.json",
+            "Microsoft.Build.Conversion.Core.dll",
+            "Microsoft.Build.dll",
+            "Microsoft.Build.Engine.dll",
+            "Microsoft.Build.Framework.dll",
+            "Microsoft.Build.Tasks.Core.dll",
+            "Microsoft.Build.Utilities.Core.dll",
+            "Microsoft.VisualStudio.Threading.dll",
+            "Microsoft.VisualStudio.Validation.dll",
```

Comment:
MS.VS.dll is library needed for JsonRpc. Microsfot.Build.* is our runtime dependency for Workspace.Desktop.dll which service hub uses.